notation.go

```diff
@@ -380,16 +380,15 @@ func Verify(ctx context.Context, verifier Verifier, repo registry.Repository, ve
 					logger.Error("Got nil outcome. Expecting non-nil outcome on verification failure")
 					return err
 				}
-
-				if _, ok := outcome.Error.(ErrorUserMetadataVerificationFailed); ok {
-					verificationFailedErr = outcome.Error
-				}
```

Comment:
This part is removed and thus this change may consider as a breaking change. Is this intended?

Reply:
Nice catch! This is not intended to be deleted.

Reply:
After the update, we could remove this part now, and it's not a breaking change.